test/libsolidity/InlineAssembly.cpp

```diff
@@ -429,6 +429,16 @@ BOOST_AUTO_TEST_CASE(revert)
 	BOOST_CHECK(successAssemble("{ revert(0, 0) }"));
 }
 
+BOOST_AUTO_TEST_CASE(returndatasize)
+{
+	BOOST_CHECK(successAssemble("{ let r := returndatasize }"));
```

Comment:
You have to look at the `SolidityInlineAssembler/Analysis` suite. The others don't assemble.